### PR TITLE
adding bootstrap navbar

### LINKED ISSUE #74: Add bootstrap nav bar (depended on Omer task)
I'm on it

### PR DESCRIPTION
**Pull request by IlanZuckerman**
fix #74

### CODE REVIEW

**Comment by IlanZuckerman** [comment]
Here is how the new page looks like:
https://imgur.com/a/jEtwiBB

**Comment by IlanZuckerman** [comment]
> Why the bootstrap code was changed?

because i couldnt find a way of combining bootstrap navbar with how our html currently looks.
However, I'm not a front end specialist. Maybe @Omeramsc can suggest another layout?

File: templates/home.html [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
+<html lang="en">
 <head>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <title>Bootstrap Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```
Comment (Omeramsc) [inline templates/home.html]: Scripts should (most of the time) be at the bottom of the body tag
The reason is, if the script is using any of the elements that exist on the HTML page, it won't find them if it is loaded before them.

File: templates/home.html [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
-<html>
-
+<html lang="en">
 <head>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <title>Bootstrap Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```
Comment (Omeramsc) [inline templates/home.html]: Notice that you replaced bootstrap 4.4.1 with an earlier version, 3.4.1.
I'm not sure if we'll feel the difference, but is there a need for it?

File: templates/home.html [hunk 3]
```diff
@@ -1,22 +1,44 @@
...
+  </div>
+</nav>
+
+<div class="container">
+  <h3>Navbar Forms</h3>
+  <p>Use the .navbar-form class to vertically align form elements (same padding as links) inside the navbar.</p>
+  <p>The .input-group class is a container to enhance an input by adding an icon, text or a button in front or behind it as a "help text".</p>
+  <p>The .input-group-btn class attaches a button next to an input field. This is often used as a search bar:</p>
+</div>
+
```
Comment (Omeramsc) [inline templates/home.html]: Again, scripts at the end of the body tag.

**Pull request merged**